src/olympia/blocklist/admin.py

```diff
@@ -550,10 +566,11 @@ def delete_queryset(self, request, queryset):
 
     def get_form(self, request, obj=None, change=False, **kwargs):
         form = super().get_form(request, obj=obj, change=change, **kwargs)
-        form.base_fields['min_version'].choices = _get_version_choices(
-            obj, 'min_version')
-        form.base_fields['max_version'].choices = _get_version_choices(
-            obj, 'max_version')
+        if 'min_version' in form.base_fields:
```

Comment:
Why is this change necessary for this patch? I don't see any changes that would cause `form.base_fields` to not have `min_version`.

Reply:
when `has_change_permission` is `False` the form is empty so `.base_fields` doesn't contain the min/max fields.